src/utils/index.ts

```diff
@@ -55,7 +55,7 @@ export const getLabelsOfSelectedItems = ({
         (item: TFlatListItem) => item[optionValue] === element
       )?.[optionLabel];
 
-      if (selectedItemLabel) {
+      if (selectedItemLabel !== '') {
```

Comment:
@azeezat , since `selectedItemLabel` could be `undefined`, I think this should be `if(selectedItemLabel){...}` because `undefined !== ''` will produce `true`

Reply:
@kofiarkoh Your observation is correct. This change is supposed to handle some edge cases. Using the Logical NOT Operator (!) negates the value of the operand. So in a scenario where the value of the label is 0 or false, the logic fails.

See example below

```js
options = [
    {label: 0, value: 0},
    {label: 1, value: false},
    {label: 2, value: 2},
  ]
```

Reply:
@azeezat , right I get it now, but how about the effect of this PR, #91  which is now enforcing labels to be  string internally?

Reply:
@kofiarkoh It doesn't affect it, it would cast it.

Reply:
okay, sounds good.